src/internal/modules/billing/lib/routing.js

```diff
@@ -23,6 +23,13 @@ const getBillingBatchRoute = (batch, opts = {}) => {
     .set('error', `/billing/batch/${id}/error`)
     .set('queued', `/billing/batch/${id}/processing?back=${opts.isBackEnabled ? 1 : 0}`)
 
+  const routeMapTptSroc = new Map()
+    .set('review', `/system/bill-runs/${id}/review`)
+
+  if (scheme === 'sroc' && type === 'two_part_tariff' && batch.status === 'review') {
+    return routeMapTptSroc.get(batch.status)
+  }
+
```

Comment:
Appreciate we can't include our new link in the existing`Map` else there would be 2 entries labelled `review`.

But as this forces us to handle this scenario outside the existing `Map` there is no need for us to create one, especially when we just have the one entry. So, I'd suggest amending the block as such

```suggestion
  if (scheme === 'sroc' && type === 'two_part_tariff' && batch.status === 'review') {
    return `/system/bill-runs/${id}/review`
  }

```

The other thing I'd suggest doing is moving this above the existing logic. In effect, we treat it as a guard clause. There is no point generating the `Map` if we never use it because we're dealing with a 2PT sroc bill run with a status of `review`.